### PR TITLE
Strong compare types

### PR DESCRIPTION
**Pull request by arv**

### CODE REVIEW

**Comment by arv** [comment]
@rossberg-chromium @Pasteuriser PTAL

I forgot to do these in the last patch.

File: src/runtime/strong.js [hunk 1]
```diff
@@ -33,79 +41,101 @@ function assertNumber(x) {
...
 }
 
 function strongMod(x, y) {
-  checkTypes(x, y);
+  checkBothNumbers(x, y);
   return x % y;
 }
 
-function strongPlus(x, y) {
-  if (typeof x === 'string' && typeof y === 'string' ||
```
Comment (Pasteuriser) [inline src/runtime/strong.js]: checkBothStringsOrBothNumbers here?

Comment (arv) [inline src/runtime/strong.js]: That is the deleted code... the new code uses checkBothStringsOrBothNumbers.

Comment (arv) [inline src/runtime/strong.js]: line 99

Comment (Pasteuriser) [inline src/runtime/strong.js]: Oops, sorry! I clearly can't read.

**Pull request closed without merging**